idaes/gas_solid_contactors/unit_models/tests/test_FB0D.py

```diff
@@ -0,0 +1,481 @@
+###############################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University,
+# West Virginia UniversityResearch Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+###############################################################################
+"""
+Tests for ControlVolumeBlockData, and for initializing the 0D fixed bed module
+
+Author: Brandon Paul
+"""
+
+import pytest
+
+from pyomo.environ import (ConcreteModel,
+                           TransformationFactory,
+                           TerminationCondition,
+                           SolverStatus,
+                           value,
+                           units as pyunits,
+                           Constraint)
+from idaes.core import (FlowsheetBlock,
+                        EnergyBalanceType)
+from idaes.core.util.model_statistics import (degrees_of_freedom,
+                                              number_variables,
+                                              number_total_constraints,
+                                              number_unused_variables,
+                                              unused_variables_set)
+from idaes.core.util.testing import initialization_tester
+from idaes.core.util.initialization import initialize_by_time_element
+from idaes.core.util import get_solver
+
+# Import FixedBed0D unit model
+from idaes.gas_solid_contactors.unit_models.fixed_bed_0D import FixedBed0D
+
+# Import property packages
+from idaes.gas_solid_contactors.properties.methane_iron_OC_reduction. \
+    gas_phase_thermo import GasPhaseParameterBlock
+from idaes.gas_solid_contactors.properties.methane_iron_OC_reduction. \
+    solid_phase_thermo import SolidPhaseParameterBlock
+from idaes.gas_solid_contactors.properties.methane_iron_OC_reduction. \
+    hetero_reactions import HeteroReactionParameterBlock
```

Comment:
You should have a test for unit consistency, and I do not see you importing the Pyomo tools for that.